services/headless-lms/server/src/domain/authorization.rs

```diff
@@ -154,11 +157,30 @@ impl<T: Responder> Responder for AuthorizedResponse<T> {
     }
 }
 
+/**  Skips the authorize() and returns AuthorizationToken, needed in functions with anonymous and test users
+
+We need skip to get all organizations on the homepage
+
+
+ async fn get_all_organizations ->
+
+ token = skip_authorize()?;
+
+token.authorized_ok(Json(courses))
+
+
+*/
 pub fn skip_authorize() -> anyhow::Result<AuthorizationToken> {
     Ok(AuthorizationToken(()))
 }
 
-/// Can user_id action the resource?
+/// Verifies that user has permission to Act on given Resource and returns an AuthorizationToken
+///
+/// let token = authorize(&mut conn, Act::Edit, Some(user.id), Res::Page(*page_id)).await?;
+///
+/// We check that the user has permission to Edit the given Page, if succesful we return authorized_ok
+///
+/// token.authorized_ok(web::Json(cms_page_info))
```

Comment:
The authorization token is the only way to return a controller result, and should only be used in controller functions that return a response to the user.